docs/msbuild/getfilehash-task.md

```diff
@@ -0,0 +1,67 @@
+---
+title: "GetFileHash  Task | Microsoft Docs"
+ms.date: "01/28/2019"
+ms.topic: "reference"
+f1_keywords:
+  - "http://schemas.microsoft.com/developer/msbuild/2003#GetFileHash"
+dev_langs:
+  - "VB"
+  - "CSharp"
+  - "C++"
+  - "jsharp"
+helpviewer_keywords:
+  - "GetFileHash task [MSBuild]"
+  - "MSBuild, GetFileHash task"
+ms.assetid: ???
```

Comment:
@Mikejo5000 should I just create a new GUID for these, or is there something else I could do to help? In general I was guessing on a lot of this metadata.

Reply:
@rainersigwald ms.assetid is used only for topics migrated from MSDN, so please remove that. Everything else looks good except f1_keywords. This metadata item is for F1 links from the IDE (if an F1 link exists) formatted like this example: vc.task.xdcmake
If there's no plan to link to the topic from the IDE, just remove f1_keywords.
Thanks!